hmt1developerexamples/src/main/java/com/realwear/hmt1developerexamples/ActionButtonActivity.kt

```diff
@@ -0,0 +1,90 @@
+/**
+ * RealWear Development Software, Source Code and Object Code
+ * (c) RealWear, Inc. All rights reserved.
+ *
+ *
+ * Contact info@realwear.com for further information about the use of this code.
+ */
+package com.realwear.hmt1developerexamples
+
+import android.app.Activity
+import android.os.Bundle
+import android.view.KeyEvent
+import android.view.View
+import android.widget.ImageView
+
+/**
+ * Activity that shows how to interact with the action button on a HMT-1 device
+ */
+class ActionButtonActivity : Activity() {
+    private val _actionButtonImageView: ImageView by lazy {
+        findViewById<View>(R.id.actionButtonImageView) as ImageView
+    }
+
+    /**
+     * Called when the activity is created
+     *
+     * @param savedInstanceState See Android docs
+     */
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.action_button_main)
+
+        // Disable the action button default behaviour
+        _actionButtonImageView.contentDescription = "hf_no_ptt_home"
+    }
+
+    /**
+     * Called when activity is resumed - See Android docs
+     */
+    override fun onResume() {
+        super.onResume()
+        _actionButtonImageView.setImageResource(R.drawable.radio_button_unchecked)
+    }
+
+    /**
+     * Called when activity is paused - See Android docs
+     */
+    override fun onPause() {
+        super.onPause()
+        finish()
+    }
+
+    /**
+     * Called when a key is pressed - See Android docs
+     *
+     * @param keyCode The value in event.getKeyCode()
+     * @param event   Description of the key event
+     * @return true if the action button was pressed, otherwise false
+     */
```

Comment:
For Kotlin code comments there is a style that we have been loosely following: https://kotlinlang.org/docs/coding-conventions.html#documentation-comments

However, Kotlin code comments start with `Returns ...` then provides a briefs explanation. 

This might be a little over-kill for your current project since it's a demo app but I thought it'd be a good style to keep in mind.

Reply:
Oh sure, I can go ahead and convert these comments as well.

Reply:
Oh I just realized this was the code comments from the original Java docs and it's nothing new. It's up to you but I don't think it's blocking for this PR.